hypothesis-python/docs/reference/strategies.rst

```diff
@@ -0,0 +1,230 @@
+.. _strategies:
+
+Strategies Reference
+====================
+
+Strategy objects tell Hypothesis what types of inputs to generate. For instance, passing ``st.lists(st.integers(), min_size=1)`` to |@given| tells Hypothesis to generate lists of integers with at least one element.
+
+Strategies can be combined using :ref:`combinator strategies <combinators>`, or modified using |strategy.filter|, |strategy.map|, or |strategy.flatmap|.
```

Comment:
```suggestion
Strategies are the way we describe values for |@given| to generate.  For instance, ``st.lists(st.integers(), min_size=1)`` tells Hypothesis to generate lists of integers with at least one element.

This reference page lists all of Hypothesis' first-party functions which return a strategy; there are also many provided by <third-party libraries>(xref), and you can define your own too.  Note that people often say "strategy" when they mean "function returning a strategy"; it's usually clear from context which they meant.

Strategies can be passed to many strategy functions as arguments, combined using :ref:`combinator strategies <combinators>`, and modified using |strategy.filter|, |strategy.map|, or |strategy.flatmap|.
```

Reply:
> Strategies can be passed to many strategy functions as arguments

I get the distinction you're making, and I think it's worth making, but this is a confusing sentence to parse imo. Bite the technical incorrectness and say "strategies can be passed to other strategies as arguments"?

I've applied a variant of this, but happy to tweak further

Reply:
I do think we need to call out both the distinction and the common sloppy usage in this explanatory header, but don't care exactly how we do that.